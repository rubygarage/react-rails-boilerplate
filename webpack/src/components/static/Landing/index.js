import React from 'react';
import { FormattedMessage } from 'react-intl';
import Header from 'containers/Header';

function Landing() {
  return (
    <main>
      <Header />
      <div className="jumbotron">
        <h1 className="display-3"> <FormattedMessage id="landing.header" /> </h1>
        <p className="lead"> <FormattedMessage id="landing.lead_message" /> </p>

        <p> <FormattedMessage id="landing.note" /> </p>
      </div>
    </main>
  );
}

export default Landing;
