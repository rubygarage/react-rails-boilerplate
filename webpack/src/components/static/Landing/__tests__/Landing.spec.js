import React from 'react';
import { shallow } from 'enzyme';
import Landing from 'components/static/Landing';

describe('<Landing />', () => {
  it('renders Landing component', () => {
    const component = shallow(<Landing />);

    expect(component).toMatchSnapshot();
  });
});
