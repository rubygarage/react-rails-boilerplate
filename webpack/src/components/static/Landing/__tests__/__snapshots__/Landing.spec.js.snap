// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders Landing component 1`] = `
<main>
  <Connect(Header) />
  <div
    className="jumbotron"
  >
    <h1
      className="display-3"
    >
       
      <FormattedMessage
        id="landing.header"
        values={Object {}}
      />
       
    </h1>
    <p
      className="lead"
    >
       
      <FormattedMessage
        id="landing.lead_message"
        values={Object {}}
      />
       
    </p>
    <p>
       
      <FormattedMessage
        id="landing.note"
        values={Object {}}
      />
       
    </p>
  </div>
</main>
`;
